README.md

```diff
@@ -154,4 +154,4 @@ Coming soon:
   * [Error](http://developer.pagerduty.com/documentation/rest/errors) handling and throttling
   * More examples
 
-Possibly also of interest is [node-pagerduty](https://github.com/Skomski/node-pagerduty) to trigger PagerDuty incidents from node.js.
\ No newline at end of file
+Possibly also of interest is [node-pagerduty](https://github.com/Skomski/node-pagerduty) to trigger PagerDuty incidents from node.js.
```

Comment:
There used to be no newline at end of file. Not sure why it's not saying so. It actually showed up in the email summary for this comment.
<img width="861" alt="screen shot 2016-09-29 at 9 31 22 am" src="https://cloud.githubusercontent.com/assets/55923/18962942/a7728d68-8627-11e6-848a-e784c5eeaf31.png">